membership/src/main/scala/zio/membership/Message.scala

```diff
@@ -0,0 +1,9 @@
+package zio.membership
+
+import zio.Chunk
+
+final case class Message(
```

Comment:
I think message should have ```correlationId``` that we can match all the message in conversation. WDYT?

Reply:
Do you mean for tracing? a combination of both should be usable for this. The idea is to have the node address + monotonically increasing counter so that every conversation has a unique id

Reply:
primarily I would like to have this for tracing. monotonically increased counter might be tricky if we will provide client library because we need to generate request id on client. this is important when someone need to debug what happen with his/her request.